### ISSUE TITLE
Nuxt JS TypeError: this.state.models[modelID] is undefined

### ISSUE THREAD

**Issue by saad4software** [IFCjs/web-ifc-viewer#173]
Following the basic sample with Nuxt JS (Vue) I have faced two issues:
1. the viewer shows the grid but doesn't show the model
![Screenshot from 2023-01-11 20-05-23](https://user-images.githubusercontent.com/85710350/211873583-bc8dfd38-013a-4252-9b66-da0df45509e7.png)
to walk around it, I used IFCLoader from  web-ifc-three

```js
import { IFCLoader } from "web-ifc-three/IFCLoader";
input.addEventListener("change",
      async (changed) => {
        const file = changed.target.files[0];
        const ifcURL = URL.createObjectURL(file);
        this.viewer.IFC.loader = new IFCLoader();
        // this.viewer.IFC.setWasmPath('../');
        await this.viewer.IFC.loadIfcUrl(ifcURL);
      },
      false
    );

```
this loaded the model, but it seems to cause the second issue
2. when trying to select surface using 
```js
    window.onmousemove = () => this.viewer.IFC.selector.prePickIfcItem();
```
I am getting this.state.models[modelID] is undefined 
![Screenshot from 2023-01-11 20-10-13](https://user-images.githubusercontent.com/85710350/211877068-2806af4e-8a80-4044-a334-0e9b1da67543.png)

although clipping surfaces works, they don't show on the grid either
please help

**Comment by agviegas** [comment]
Hey, can you please send us a link with a minimal application where we can reproduce this? Thanks!

**Comment by saad4software** [comment]
Yes of course
https://github.com/saad4software/nuxt_ifc_viewer

```
npm install
npm run dev
```
please notice that I have added multiple wasm files, it can be improved, but I tried to add them where ever the library asks
and you can reproduce the first issue by commenting line 33 in pages/index.vue

**Comment by agviegas** [comment]
It's never recommended to install both `web-ifc-three` and `web-ifc-viewer`, as the latter contains the first and it can cause version conflict. I was able to solve this by `npm uninstall web-ifc-three`, deleting the `node_modules` folder, executing `npm install` and then updating the wasm files of the folder of the image below. 

I noticed multiple copies of the wasm files in the folder structure under `static`. I don't have previous experience with this technology, so I can't say if this is expected, but I thought it was worth mentioning it.

![image](https://user-images.githubusercontent.com/56475338/212614803-f6874532-13e3-4745-985a-4c3c0880e7cf.png)
